src/schema/schema.graphql

```diff
@@ -2237,6 +2265,12 @@ input ExternalAccountModifyAttrsInput {
 	port: String
 	smtpPort: String
 }
+input EnableTwoFactorAuthInput {
+	authToken: [TwoFactorAuthTokenInput]
```

Comment:
do we need to send authtoken explicitly? that should already be passed as ZM_AUTH_TOKEN cookie

Reply:
done